### PR TITLE
Add gpt_for_sequence_classification benchmark

### PR DESCRIPTION
**Pull request by FrostML**
添加 gpt_for_sequence_classification benchmark 示例。

### CODE REVIEW

File: frame_benchmark/docker_images.yaml [hunk 1]
```diff
@@ -47,6 +47,7 @@ pytorch:
    segformer_b0: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda11.2-cudnn8
    xlnet: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda11.2-cudnn8
    gpt3: iregistry.baidu-int.com/paddlecloud/base-images:paddlecloud-ubuntu18.04-gcc8.2-cuda11.1-cudnn8
+   gpt_for_sequence_classification: registry.baidubce.com/paddlepaddle/paddle:2.4.1-gpu-cuda11.7-cudnn8.4-trt8.4
```
Comment (mmglove) [inline frame_benchmark/docker_images.yaml:50]: 镜像请更新为：iregistry.baidu-int.com/paddlecloud/base-images:paddlecloud-ubuntu18.04-gcc8.2-cuda11.7-cudnn8.4.1-nccl2.12.12

**Review by mmglove — COMMENTED** [review]
请贴上3次稳定性自测数据

File: frame_benchmark/pytorch/dynamic/PaddleNLP/scripts/gpt_for_sequence_classification/benchmark_common/analysis_log.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    base_batch_size, fp_item, run_mode = args.model_name.split("_")[-3:]
+    base_batch_size = int(base_batch_size.replace("bs",""))
+
+    run_info = {    
+                "model_branch": os.getenv('model_branch'),
+                "model_commit": os.getenv('model_commit'),
+                "model_name": args.model_name,
+                "batch_size": base_batch_size,
+                "fp_item": fp_item,
+                "run_process_type": args.run_process_type,
```
[the remark this answers is not in the document]
Comment (FrostML) [reply frame_benchmark/pytorch/dynamic/PaddleNLP/scripts/gpt_for_sequence_classification/benchmark_common/analysis_log.py]: Done. Thanks.

File: frame_benchmark/pytorch/dynamic/PaddleNLP/scripts/gpt_for_sequence_classification/benchmark_common/PrepareEnv.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+echo "******prepare benchmark start************"
+
+echo "https_proxy $HTTPS_PRO" 
+echo "http_proxy $HTTP_PRO" 
+export https_proxy=$HTTPS_PRO
+export http_proxy=$HTTP_PRO
+export no_proxy=localhost,bj.bcebos.com,su.bcebos.com 
+
+# wget ${FLAG_TORCH_WHL_URL}
```
[the remark this answers is not in the document]
Comment (FrostML) [reply frame_benchmark/pytorch/dynamic/PaddleNLP/scripts/gpt_for_sequence_classification/benchmark_common/PrepareEnv.sh]: Done. Thanks.

File: frame_benchmark/docker_images.yaml (hunk 1 above)
Comment (FrostML) [reply frame_benchmark/docker_images.yaml]: Done. Thanks.

**Comment by FrostML** [comment]
pytorch_gpt_for_sequence_classification_bs8_fp16_DP_N1C1 | 45877.365 | 45692.077 | 45597.493028
-- | -- | -- | --
pytorch_gpt_for_sequence_classification_bs8_fp16_DP_compile_N1C1 | 102928.79669 | 102334.122 | 101095.86336
pytorch_gpt_for_sequence_classification_bs8_fp32_DP_N1C1 | 39724.18073 | 39891.7 | 39680.1178
pytorch_gpt_for_sequence_classification_bs8_fp32_DP_compile_N1C1 | 68249.60426 | 67448.96 | 67324.12886
pytorch_gpt_for_sequence_classification_bs8_fp16_DP_N1C8 | 365394.15802 | 366118.608 | 366536.53844
pytorch_gpt_for_sequence_classification_bs8_fp16_DP_compile_N1C8 | 816841.3705 | 815025.032 | 805773.79415
pytorch_gpt_for_sequence_classification_bs8_fp32_DP_N1C8 | 316764.302 | 317370.536 | 318457.47162
pytorch_gpt_for_sequence_classification_bs8_fp32_DP_compile_N1C8 | 545174.73442 | 539466.864 | 537766.5816

**Review by mmglove — APPROVED** [review]

**Pull request merged**